tests/app/dao/test_ft_billing_dao.py

```diff
@@ -0,0 +1,233 @@
+from decimal import Decimal
+
+from datetime import datetime, timedelta
+from freezegun import freeze_time
+
+from app import db
+from app.dao.fact_billing_dao import (
+    fetch_annual_billing_for_year, fetch_billing_data_for_day, get_rates_for_billing,
+    get_rate
+)
+from app.models import FactBilling
+from app.utils import convert_utc_to_bst
+from tests.app.db import (
+    create_ft_billing,
+    create_service,
+    create_template,
+    create_notification,
+    create_rate,
+    create_letter_rate
+)
+
+
+def test_fetch_billing_data_for_today_includes_data_with_the_right_status(notify_db_session):
+    service = create_service()
+    template = create_template(service=service, template_type="email")
+    for status in ['delivered', 'sending', 'temporary-failure', 'created', 'technical-failure']:
+        create_notification(template=template, status=status)
+
+    today = convert_utc_to_bst(datetime.utcnow())
+    results = fetch_billing_data_for_day(today)
+    assert len(results) == 1
+    assert results[0].notifications_sent == 3
```

Comment:
This feels a bit indirect since we're not checking the statuses that end up in the result. Maybe split this in two parts - one tat checks the count after creating all successful statuses and one that checks for 0 after creating all filtered ones.